app/Http/Controllers/PlayAreaController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Auth; 
+use App\Question;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Redirect;
+
+class PlayAreaController extends Controller
+{
+    public function showQuestion()
+    {
+        $user = Auth::user();
+        $question = Question::where('level', $user->current_level)->first();
+        return view('playarea', compact('question'));
+    }
+
+    public function evaluateAnswer(Request $request)
```

Comment:
here, method name `evaluateAnswer` is very verbose and very specific about what this method does, this should be something generic.

We may not be just evaluating, we would be incrementing user's level, and award some score, and maybe something more in future.

So, for now, just rename it to `postAnswer`, this doesn't suggest we are essentially evaluating user's answer. This suggests a broader action i.e. posting answer, you maybe doing ten different things when someone postsAnswer.